### PR TITLE
tweak URL for payment-instructions.json in README

### PR DESCRIPTION
**Pull request by whit537**
Reticketed from https://github.com/gratipay/gratipay.com/pull/3909#issuecomment-182063339.

cc: @aandis

### CODE REVIEW

**Comment by aandis** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -523,11 +523,11 @@ E.g.:
 Request
 
 ```
-curl -L https://gratipay.com/username/payment-instructions.json \
```
Comment (rohitpaulk) [inline README.md]: The heading of this section should be changed too ( **/`%username`/payment-instructions.json**)

Comment (aandis) [inline README.md]: Good idea. Should also change `/%username/public.json` to `/~username/public.json` for uniformity.

Comment (whit537) [inline README.md]: Done in a6da087a350233a64f8dd5c6c2a0e9f55c9b1b69.

**Comment by whit537** [comment]
@aandis Is this going to be the first PR you merge? :smile:

**Comment by aandis** [comment]
:blush:

**Comment by whit537** [comment]
!m @aandis :dancer:

**Pull request merged**